x/mongo/driver/mongocrypt/mongocrypt.go

```diff
@@ -26,7 +26,8 @@ import (
 )
 
 type MongoCrypt struct {
-	wrapped *C.mongocrypt_t
+	wrapped      *C.mongocrypt_t
+	kmsProviders *bsoncore.Document
```

Comment:
Is there a reason to use a `*bsoncore.Document` instead of a `bsoncore.Document`?